state/pool.go

```diff
@@ -181,9 +177,6 @@ func OpenStatePool(args OpenParams) (_ *StatePool, err error) {
 		RestartDelay: time.Second,
 		Clock:        args.Clock,
 	})
-	pool.hubUnsubFn = pool.hub.Subscribe(watcher.TxnWatcherStarting, func(string, interface{}) {
```

Comment:
I suspect this was caused by the fact we weren't catching every location that the unsubscribe could have been called. Either way, I'm happy that we've removed this, as it's not used. Seems like originally it was created because they could.